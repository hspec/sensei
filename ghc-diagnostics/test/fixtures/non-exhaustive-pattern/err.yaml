version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/non-exhaustive-pattern/Foo.hs
  start:
    line: 3
    column: 9
  end:
    line: 4
    column: 14
severity: Error
code: 62161
message:
- |-
  Pattern match(es) are non-exhaustive
  In a case alternative:
      Patterns of type `Maybe Int' not matched: Nothing
hints: []
reason:
  flags:
  - incomplete-patterns
