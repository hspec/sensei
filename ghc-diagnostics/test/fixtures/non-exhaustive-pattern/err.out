test/fixtures/non-exhaustive-pattern/Foo.hs:3:9: error: [GHC-62161] [-Wincomplete-patterns, Werror=incomplete-patterns]
    Pattern match(es) are non-exhaustive
    In a case alternative:
        Patterns of type `Maybe Int' not matched: Nothing

